### DISCUSSION TITLE
polling_interval

### DISCUSSION THREAD

**Discussion by Kazturin** [filamentphp/filament#7382] [Help]
Hi.
In the notifications config file, I set the 'polling_interval' values ​​to null. But still every 30 seconds laravel makes a request

**Comment by pboivin** [comment]
Can you share your config file?

**Reply by Kazturin** [reply]
![Screenshot_1](https://github.com/filamentphp/filament/assets/71196888/38db5e04-251c-401b-89ff-d0ae59a1b641)
notification.php

**Reply by pboivin** [reply]
I think the key should be `database_notifications` instead of `database`

**Reply by Kazturin** [reply]
Thanks for your answer but it didn't help
